### ISSUE TITLE
Adjust ammo rules?

### ISSUE THREAD

**Issue by DukeVenator** [wintersleepAI/foundry-swnr#56]
Love the sheet but looking at potentially adding multiple fire mods for ammo use and hit modifiers. 

Any idea how I would implement some custom fire rate or ammo use a round?

**Comment by DukeVenator** [comment]
Looking at this the easiest way would be to modify the weapon's code to remove ammo if a fire rate modifier is defined and then add a box somewhere to input a ammo fire rate.

**Comment by wintersleepAI** [comment]
I don't think this is something that could be easily added, one way might be for a popup with a special click+button pres. the burst mode is in the code rather statically.
